packages/did-auth-siop-op-authenticator/README.md

```diff
@@ -0,0 +1,214 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>DID Auth SIOP OP Authenticator (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in every early development. Breaking changes without notice will happen at this point!**
+
+---
+
+A Veramo operating party authentication plugin using the [Self Issued OpenID Provider v2 (SIOP)](https://github.com/Sphereon-Opensource/did-auth-siop) authentication library for having clients / people conforming to
```

Comment:
operating party? I think you are confusing a Self issued OpenID Provider (SIOP) with a Relying Party. Currently the plugin is mainly involved on the SIOP side of things.
Given the complete sentence I think it makes most sense to just have 'A Veramo authentication ....'

Reply:
ye makes sense, changed